predictionguard/client.py

```diff
@@ -525,9 +539,14 @@ def _generate_embeddings(self, model, input):
                         )
                     os.remove(temp_image)
 
+                elif bool(data_uri_pattern.match(item["image"])):
```

Comment:
No need to convert to bool, if it matches it will be truthy
```suggestion
                elif data_uri_pattern.match(item["image"]):
```